README.rst

```diff
@@ -103,130 +103,75 @@ Functionality Overview
+
+-   **North-south load balancing** supports XDP for high-throughput scenarios and `layer 4 load balancing <https://cilium.io/use-cases/load-balancer/>`_ including Direct Server Return (DSR), and Maglev consistent hashing.
+
+**Cluster Mesh**
+----------------
+
+Cilium `Cluster Mesh <https://cilium.io/use-cases/cluster-mesh/>`_ enables secure, seamless connectivity across multiple Kubernetes clusters. For operators running hybrid or multi-cloud environments, Cluster Mesh ensures a consistent security and connectivity experience.
+
+-   **Global service discovery**: Workloads across clusters can discover and connect to services as if they were local.
+
+-   **Unified identity model**: Security policies are enforced based on identity, not IP address, across all clusters.
+
+-   **High availability**: Supports redundant paths and load balancing across clusters.
+
+**Network Policy**
+------------------
+
+Cilium `Network Policy <https://cilium.io/use-cases/network-policy/>`_ provides identity-aware enforcement across L3–L7. Typical container firewalls secure workloads by filtering on source IP addresses and destination ports. This concept requires the firewalls on all servers to be manipulated whenever a container is started anywhere in the cluster.
+
+In order to avoid this situation which limits scale, Cilium assigns a security identity to groups of application containers which share identical security policies. The identity is then associated with all network packets emitted by the application containers, allowing to validate the identity at the receiving node.
+
+-   **Identity-based security** removes reliance on brittle IP addresses.
+
+-   **L3/L4 policies** restrict traffic based on labels, protocols, and ports.
+
+-   **L7-aware policies** allow filtering by HTTP method, URL path, gRPC call, Kafka topic, and more:
+
+    -   Example: Allow only GET requests to /public/.\*.
+    -   Enforce the presence of headers like X-Token: \[0-9\]+.
+    -   Permit one service to produce to Kafka topic topic1 and another to consume it.
+
+CIDR-based egress and ingress policies are also supported for controlling access to external IPs, ideal for integrating with legacy systems or regulatory boundaries.
+
+**Service Mesh**
+----------------
+
+With Cilium `Service Mesh <https://cilium.io/use-cases/service-mesh/>`_, operators gain the benefits of service mesh, encryption, observability, access control, without the cost and complexity of traditional proxy-based designs. Key features include:
+
+-   **Mutual authentication** with automatic identity-based encryption between workloads.
+
+-   **L7-aware policy enforcement** for security and compliance.
+
+-   **Deep integration with the** `Kubernetes Gateway API <https://cilium.io/use-cases/gateway-api/>`_**:** Acts as a Gateway API-compliant data plane, allowing you to declaratively manage ingress, traffic splitting, and routing behavior using Kubernetes-native CRDs.
```

Comment:
Looks like formatting breaks on this line, in particular on
```
`Kubernetes Gateway API <https://cilium.io/use-cases/gateway-api/>`_**:**
```

Maybe try removing the `**` markers right after the link, to see if that makes a difference?